### PR TITLE
Front Page

### PR DESCRIPTION
**Pull request by anantjain6**
Patch #19

### CODE REVIEW

**Comment by anantjain6** [comment]
@ankitjain28may Sir, plz review

File: resources/views/dashboard.blade.php [hunk 1]
```diff
@@ -106,275 +64,44 @@
...
                 <div class="border">
                   <div class="icon">
-                    <i class="fa fa-comments-o fa-2x"></i>
+                    <i class="fa fa-cogs fa-2x"></i>
                   </div>
                 </div>
-                <h3>Coordinate</h3>
-                <p>Fifth abundantly made Give sixth hath. Cattle creature i be don't them behold green moved fowl Moved life us beast good yielding. Have bring.</p>
+                <h3>Test</h3>
+                <p>Our system will test your code with verious test case with all the possiblities.</p>
```
Comment (ankitjain28may) [inline resources/views/dashboard.blade.php:86]: typos 'various test cases' and possibilities

File: resources/views/dashboard.blade.php [hunk 2]
```diff
@@ -106,275 +64,44 @@
...
               <div class="home box-simple">
                 <div class="border">
                   <div class="icon">
-                    <i class="fa fa-cogs fa-2x"></i>
+                    <i class="fa fa-globe fa-2x"></i>
                   </div>
                 </div>
-                <h3>Test</h3>
-                <p>Fifth abundantly made Give sixth hath. Cattle creature i be don't them behold green moved fowl Moved life us beast good yielding. Have bring.</p>
+                <h3>Compet</h3>
```
Comment (ankitjain28may) [inline resources/views/dashboard.blade.php:96]: Typo error Compete

**Review by ankitjain28may — CHANGES_REQUESTED** [review]
You have removed the extra code, I haven't checked it by running locally, Will check soon. Try to add some code like displaying latest events going on.

**Comment by ankitjain28may** [comment]
No activity since last review, Closing this.

**Pull request closed without merging**